types/zpool.nix

```diff
@@ -55,7 +66,7 @@
       default = _: ''
         readarray -t zfs_devices < <(cat "$disko_devices_dir"/zfs_${config.name})
         zpool create ${config.name} \
-          ${config.mode} \
+          -R ${config.mountRoot} ${config.mode} \
```

Comment:
`man zpool-create` suggests that implicitly sets `cachefile=none` and `altroot` properties on the pool, even if no alt-root is used. Do we want this or should we use `lib.optionalString` here?

Reply:
I just set ``/`` to keep backwards compatibility but it doesn't make much sense as it will most likely overwrite the system that the command is run from.

Reply:
Ah yes, did a bit more docs reading, `cachefile=none` seems harmless.

I'd still prefer to only set `-R` if actually used, but don't care too much :)

Reply:
It's important to have -R on create too, otherwise any datasets that have `option.mountpoint` set will be mounted when they are created. If altroot is not set, this can shadow existing mountpoints like `/`, which would be fatal.